src/validation.ts

```diff
@@ -269,4 +334,42 @@ export class Validation {
     return this;
   }
 
+  /**
+   * Validate user feedback
+   *
+   * @param data
+   * @returns
```

Comment:
You can remove these empty lines (or add descriptions for the param/returns). Lots of the old code seems to have empty lines like this, but I think it's best not to copy this practice for new code!